pkg/services/authz/zanzana/common/tuple.go

```diff
@@ -113,7 +113,7 @@ var VerbMapping = map[string]string{
 	utils.VerbUpdate:           RelationUpdate,
 	utils.VerbPatch:            RelationUpdate,
 	utils.VerbDelete:           RelationDelete,
-	utils.VerbDeleteCollection: RelationDelete,
+	utils.VerbDeleteCollection: RelationDelete, // <<< IT MEANS you can delete *EVERYTHING*... i think we want it to be rare
 	utils.VerbGetPermissions:   RelationGet,
 	utils.VerbSetPermissions:   RelationDelete,
```

Comment:
@kalleep @chaudyg  -- these mappings do not seem right now that we have an explicit Permissions relation, correct?

Reply:
No that is wrong, will fix it